projects/plugins/jetpack/tools/build-block-manifest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Build script to generate a PHP manifest file containing block metadata from block.json files.
+ * This reduces filesystem reads and JSON parsing at runtime.
+ */
+
```

Comment:
Some of the WordPress-specific sniffs don't make sense for a build script like this.
```suggestion

// phpcs:disable WordPress.WP.AlternativeFunctions, WordPress.PHP.DevelopmentFunctions.error_log_var_export

```